git.go

```diff
@@ -123,6 +124,17 @@ func (g *GitClient) Merge(sha string) error {
 	return nil
 }
 
+// Rebase ...
+func (g *GitClient) Rebase(baseSha string, headSha string) error {
```

Comment:
`baseSha` should probably be renamed to `baseRef` as this is the value we are passing as the first argument in `in.go`?